From 04ddad4c3916d17887d9b9a9e2976ba963bd1c4e Mon Sep 17 00:00:00 2001
From: Josef Schlehofer <pepe.schlehofer@gmail.com>
Date: Sun, 18 Apr 2021 16:46:14 +0200
Subject: [PATCH] host-pip-requirements: add foris-plugins-distutils

It required to upload it on PyPI, which is done.
---
 lang/python/host-pip-requirements/foris-plugins-distutils.txt | 2 ++
 1 file changed, 2 insertions(+)
 create mode 100644 lang/python/host-pip-requirements/foris-plugins-distutils.txt

diff --git a/lang/python/host-pip-requirements/foris-plugins-distutils.txt b/lang/python/host-pip-requirements/foris-plugins-distutils.txt
new file mode 100644
index 000000000..2b6f80be8
--- /dev/null
+++ b/lang/python/host-pip-requirements/foris-plugins-distutils.txt
@@ -0,0 +1,2 @@
+foris-plugins-distutils==0.31.1 --hash=sha256:58fdcb842f4ad8b3138beb281a3b06bebc8e483c7df6fef236fa6973a7973414
+
-- 
2.25.1

